apis/s3/src/main/java/org/jclouds/s3/S3Client.java

```diff
@@ -260,7 +264,7 @@ DeleteResult deleteObjects(@Bucket @EndpointParam(parser = AssignCorrectHostname
     * @return ETag of the content uploaded
     * @throws org.jclouds.http.HttpResponseException
     *            if the conditions requested set are not satisfied by the object on the server.
-    * @see org.jclouds.s3.domain.CannedAccessPolicy#PRIVATE
+    * @see CannedAccessPolicy#PRIVATE
```

Comment:
There are a lot of noise changes in this PR -- can you revert them?

Reply:
All the noise changes will be corrected with bucket configuration PR